### ISSUE TITLE
High CPU load

### ISSUE THREAD

**Issue by plokoon200** [Cazadorsniper/PO3#538]
Havent seen a similar issue before so here it goes
Pack Version: 3.0.41
Online Multiplayer (mainly server issue
Description: High CPU load when using the Mechanical user in combination with sieves
Steps to reproduce: Have automatic sieve setup (machanical user and sieves) running on server startup 

Had no crash i rather have an inconvenient "fix"

I was experiencing very high cpu load on my server (95+%) and couldnt figure out where it came from i tried disabling a few things in my world and appearently turning the mechanical users off and on again helped alot (down to 50-60% load) now iam wondering if that could be due to a bug or if its just my server. Either way hope this can help in some way.

TLDR: high cpu load? try turning the Mechanical users off and on again.

**Comment by timmbobb** [comment]
You could just use the water wheel. It automatically sieves without the need of a mechanical user.

**Comment by IgnisFlamme** [comment]
Yea, there seems to be a bug with the mechanical user interacting with the sieves. This isn't the first time too. It happens in a lot of modpacks so this really isnt "fixable" by Cazador its more of a mods not playing well with each other. As timmbobb said above use the auto-sieve and water wheels it shows in Alpha section of the Quests
